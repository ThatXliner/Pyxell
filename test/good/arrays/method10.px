
a = [1..10^^6]
print a.find(0), (a*2).find(723641)
print a.count(-2), (a*5).count(9358)