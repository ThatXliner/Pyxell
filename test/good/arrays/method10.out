null 723640
0 5
